mne_bids/mne_bids.py

```diff
@@ -107,26 +115,55 @@ def _events_tsv(events, raw, fname, event_id, verbose):
         The data as MNE-Python Raw object.
     fname : str
         Filename to save the events.tsv to.
-    event_id : dict | None
-        Dictionary mapping a brief description key to an event id (value). For
-        example {'Go': 1, 'No Go': 2}.
+    trial_type : dict | None
+        Dictionary mapping a brief description key to an event id (event_value)
+        For example {'Go': 1, 'No Go': 2}. This corresponds to the 'trial_type'
+        column in BIDS.
+    event_type : dict | None
+        Dictionary mapping a brief description key to an event id (event_value)
+        if this is different from 'trial_type'. Corresponding to the
+        'event_type' column in BIDS.
```

Comment:
is this a new column in BIDS-EEG? It doesn't exist in the main specification ...